Samples/Emoji8/UWP/cs/Emoji8/Emoji8.csproj

```diff
@@ -17,14 +17,16 @@
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
-    <PackageCertificateKeyFile>Emoji8_TemporaryKey.pfx</PackageCertificateKeyFile>
+    <PackageCertificateKeyFile>

```

Comment:
the certificates used to have names - do they not need those anymore?

Reply:
This reformatting is how visual studio updated the project files automatically. It seems like it is now inferring the pfx file just by it being included in the output directory. My thought is to not mess with how visual studio is doing it, and that in more recent visual studio versions they must have changed it. I will run a pipeline once before merging to ensure it still works remotely though.